.storybook/preview.ts

```diff
@@ -25,5 +25,15 @@ export const decorators = [
     context.canvasElement.style = style
     return story()
   },
-  () => Layout
+  () => Layout,
+  withThemeByDataAttribute({
+    themes: {
+      system: 'system',
+      light: 'light',
+      dark: 'dark'
+    },
+    defaultTheme: 'light',
```

Comment:
```suggestion
    defaultTheme: 'system',
```

I think this should default to the system theme